addons/web/models/res_partner.py

```diff
@@ -27,12 +27,12 @@ def _build_vcard(self):
         vcard = vobject.vCard()
         # Name
         n = vcard.add('n')
-        n.value = vobject.vcard.Name(family=self.name)
+        n.value = vobject.vcard.Name(family=self.name or self.complete_name)
```

Comment:
If we really want to be sure ...
```suggestion
        n.value = vobject.vcard.Name(family=self.name or self.complete_name or '')
```